src/Workspaces/CSharp/Portable/Classification/ClassificationHelpers.cs

```diff
@@ -162,12 +191,61 @@ private static string GetClassificationForIdentifier(SyntaxToken token)
             {
                 return ClassificationTypeNames.Keyword;
             }
+            else if (token.Parent is ExternAliasDirectiveSyntax externAliasDirectiveSyntax && externAliasDirectiveSyntax.Identifier == token)
+            {
+                return ClassificationTypeNames.NamespaceName;
+            }
+            else if (token.Parent is LabeledStatementSyntax labledStatementSyntax && labledStatementSyntax.Identifier == token)
```

Comment:
&& part shouldn't be necessary.  But if you're tyrin to be very paranoid, that's ok :)